build_resume.py

```diff
@@ -58,7 +50,8 @@ def update_values(dict_values):
     build_dir = os.path.join(os.getcwd(), "build")
     tex_template_filepath = os.path.join(build_dir, "resume.tex")
 
-    filename = generate_filename(dict_values['FULL~NAME'].split()[-1])
+    last_name = dict_values['FULL~NAME'].split()[-1]
+    filename = "Resume{}".format("_"+last_name if last_name else "")
```

Comment:
1) it's not ideal to have to know about the key here (`dict_values['FULL~NAME']`) :/ 
     - not sure of the best remedy for that and it's _really_ not a big deal.
2) but [last names don't always work like that](http://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/)! 
    - good point. 👍  but mine does and it's not being used externally yet.
    - happy to look at this again if someone expresses interest in it. even happier to take a PR ! : ]